app/models/concerns/hyrax/ability.rb

```diff
@@ -49,6 +49,10 @@ def can_create_any_work?
       end && source_ids_for_deposit.any?
     end
 
+    def can_deposit_into_collection?(collection)
+      Hyrax::Collections::PermissionsService.can_deposit_in_collection(user: current_user, collection: collection)
+    end
```

Comment:
This ability exists in [app/models/concerns/hyrax/ability/collection_ability.rb](https://github.com/samvera/hyrax/blob/collections-sprint/app/models/concerns/hyrax/ability/collection_ability.rb#L15-L17).  You can check it with...  `if can? :deposit, collection`